src/pages/groupComparison/AlterationEnrichments.tsx

```diff
@@ -46,23 +50,30 @@ export default class AlterationEnrichments extends React.Component<
         render: () => {
             let headerName = GENOMIC_ALTERATIONS_TAB_NAME;
             return (
-                <AlterationEnrichmentContainer
-                    data={this.props.store.alterationsEnrichmentData.result!}
-                    groups={
-                        this.props.store.alterationsEnrichmentAnalysisGroups
-                            .result
-                    }
-                    alteredVsUnalteredMode={false}
-                    headerName={headerName}
-                    patientLevelEnrichments={
-                        this.props.store.usePatientLevelEnrichments
-                    }
-                    onSetPatientLevelEnrichments={
-                        this.props.store.setUsePatientLevelEnrichments
-                    }
-                    store={this.props.resultsViewStore}
-                    comparisonStore={this.props.store}
-                />
+                <div data-test="GroupComparisonAlterationEnrichments">
+                    <AlterationEnrichmentContainer
+                        data={
+                            this.props.store.alterationsEnrichmentData.result!
+                        }
+                        groups={
+                            this.props.store.alterationsEnrichmentAnalysisGroups
+                                .result
+                        }
+                        alteredVsUnalteredMode={false}
+                        headerName={headerName}
+                        patientLevelEnrichments={
+                            this.props.store.usePatientLevelEnrichments
+                        }
+                        onSetPatientLevelEnrichments={
+                            this.props.store.setUsePatientLevelEnrichments
+                        }
+                        store={this.props.resultsViewStore}
+                        comparisonStore={this.props.store}
+                        dashToRight={this.useInlineTypeSelectorMenu}
+                    >
+                        {this.props.children}
```

Comment:
did you mean to pass children here?  I don't see a <AlterationEnrichments> that has an children

Reply:
Indeed. Removed.